libsolidity/codegen/ExpressionCompiler.cpp

```diff
@@ -1172,6 +1211,13 @@ bool ExpressionCompiler::visit(MemberAccess const& _memberAccess)
 				"Tried to use .push() on a non-dynamically sized array"
 			);
 		}
+		else if (member == "pop")
+		{
+			solAssert(
+				type.isDynamicallySized() && type.location() == DataLocation::Storage,
+				"Tried to use .pop() on a non-dynamically sized array"
```

Comment:
"statically sized"?

Reply:
That reads better. Should I change the "push" Assert as well?

Reply:
Ah I see, the text is similar above :-)